rust/android/rustNDKDocker/dockerfile

```diff
@@ -0,0 +1,56 @@
+ARG rust_ver=1.56
+FROM rust:${rust_ver}-slim-bullseye
```

Comment:
Why are we using 'slim'?  From this doc it seems like this is a space optimization?

[This image does not contain the common packages contained in the default tag and only contains the minimal packages needed to run rust. Unless you are working in an environment where only the rust image will be deployed and you have space constraints, we highly recommend using the default image of this repository.](https://registry.hub.docker.com/_/rust/)

Reply:
I don't remember what directed my decision, I believe it was over dependences and other containers I saw and got ideas from